src/backend/src/prisma/schema.prisma

```diff
@@ -555,3 +565,57 @@ model Manufacturer {
   dateDeleted   DateTime?
   materials     Material[]
 }
+
+enum Design_Review_Status {
+  UNCONFIRMED
+  CONFIRMED
+  SCHEDULED
+  DONE
+}
+
+enum Design_Review_Team {
+  ELECTRICAL
+  SOFTWARE
+  MECHANICAL
+}
+
+model Design_Review {
+  designReviewId   Int                  @id @default(autoincrement())
+  dateScheduled    DateTime             @db.Timestamp(3)
```

Comment:
ok so, you should add a field called meetingTime, that's an int value that functions the same as availability does below. (see if you can add some comments in the schema ab it) and you should make this field @db.Date, because we just wanna store the date, not the time, to avoid any datetime fuckery, see this for more details: https://nerdocs.atlassian.net/wiki/spaces/NER/pages/217448455/Datetime+Documentation